packages/asset-swapper/src/types.ts

```diff
@@ -131,7 +131,13 @@ export interface ForwarderExtensionContractOpts {
     feeRecipient: string;
 }
 
-export interface AffiliateFee {
+export enum AffiliateFeeType {
+    PercentageFee,
```

Comment:
Wondering if it's cleaner/clearer to also have an explicit `None` type.

Reply:
right, I think it does and would make "do we have an affiliateFee" checks easier